tools/cldr-code/src/main/java/org/unicode/cldr/util/ExtraPaths.java

```diff
@@ -278,19 +283,14 @@ private static void addDayPlurals(Set<String> toAddTo, String localeID) {
 
     private static void addCurrencies(
             Set<String> toAddTo, Set<SupplementalDataInfo.PluralInfo.Count> pluralCounts) {
-        for (String code : supplementalData.getBcp47Keys().getAll("cu")) {
-            String currencyCode = code.toUpperCase();
-            toAddTo.add(
-                    "//ldml/numbers/currencies/currency[@type=\"" + currencyCode + "\"]/symbol");
-            toAddTo.add(
-                    "//ldml/numbers/currencies/currency[@type=\""
-                            + currencyCode
-                            + "\"]/displayName");
-            if (!pluralCounts.isEmpty()) {
+        // This code is locale-dependent due to pluralCounts. Compare addPaths(NameType.CURRENCY)
+        // and addPaths(NameType.CURRENCY_SYMBOL) above, which are locale-independent.
+        if (!pluralCounts.isEmpty()) {
+            for (String code : StandardCodes.make().getGoodAvailableCodes(currency)) {
```

Comment:
`getGoodAvailableCodes` returns the same set of codes as `supplementalData.getBcp47Keys().getAll("cu")` aside from being already uppercase